app/src/main/java/se/inera/intyg/minaintyg/error/ErrorController.java

```diff
@@ -0,0 +1,30 @@
+package se.inera.intyg.minaintyg.error;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import se.inera.intyg.minaintyg.error.dto.LogErrorRequestDTO;
+import se.inera.intyg.minaintyg.error.service.LogErrorService;
+import se.inera.intyg.minaintyg.error.service.dto.LogErrorRequest;
+
+@Slf4j
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/error")
+public class ErrorController {
```

Comment:
Is LogErrorController a better name for this controller? I guess it depends if it will handle other things than logging in the future.

Reply:
I think LogErrorController is too specific for a controller but both work.

Reply:
ErrorController works fine for me! :)

Reply:
I think you are on something here... I believe it should be called LogController to say what it does. Then it makes very much sense to call the endpoint /error. So /api/log/error.